test/e2e/framework/test_context.go

```diff
@@ -349,24 +349,6 @@ func RegisterClusterFlags(flags *flag.FlagSet) {
 	flags.DurationVar(&nodeKiller.SimulatedDowntime, "node-killer-simulated-downtime", 10*time.Minute, "A delay between node death and recreation")
 }
 
-// RegisterNodeFlags registers flags specific to the node e2e test suite.
-func RegisterNodeFlags(flags *flag.FlagSet) {
```

Comment:
Are we possibly breaking callers of the framework from using flags even though they're not used?

Reply:
Sorry if I misunderstand your point.
All callers of this function `RegisterNodeFlags()` seems only one `test/e2e_node/e2e_node_suite_test.go` in Kubernetes related repositories as https://cs.k8s.io/?q=RegisterNodeFlags&i=nope&files=&repos=
Then I am guessing this change would not break any callers.

Reply:
Sorry if I wasn't clear -- the flags are still currently registered as part of the core framework though, so do any tests depend on the TestContext fields that get populated here?

Reply:
I think I got your point.
After investigating all flags which are registered in this function, I need to investigate more related to `TestContext.NodeName`, `TestContext.PrepullImages` and `TestContext.SystemSpecName`
So I'd like to take more time for these flags.

/hold

Reply:
@andrewsykim I've investigated how to use these flags as the following and it is fine to move all flags into e2e_node test because I see all flags are e2e_node specific.